waddrmgr/manager_test.go

```diff
@@ -9,6 +9,7 @@ import (
 	"encoding/hex"
 	"errors"
 	"fmt"
+	"github.com/btcsuite/btcutil/hdkeychain"
```

Comment:
nit: import needs to be grouped with the other external imports

Reply:
Fixed.